rotkehlchen/tests/api/test_assets.py

```diff
@@ -262,3 +262,133 @@ def test_ignored_assets_endpoint_errors(rotkehlchen_api_server_with_exchanges, m
         )
         # Check that assets did not get modified
         assert set(rotki.data.db.get_ignored_assets(cursor)) >= expected_tokens
+
+
+def test_get_all_assets(rotkehlchen_api_server):
+    """Test that check fetching all assets returns a paginated result."""
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'allassetsresource',
+        ),
+        json={
+            'limit': 20,
+            'offset': 0,
+            'asset_type': 'fiat',
+            'order_by_attributes': ['name'],
+            'ascending': [True],
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert len(result['entries']) == 20
+    assert 'entries' in result
+    assert 'entries_found' in result
+    assert 'entries_total' in result
+    assert 'entries_limit' in result
+
+    # check that providing multiple order_by_attributes fails
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'allassetsresource',
+        ),
+        json={
+            'limit': 20,
+            'offset': 0,
+            'asset_type': 'fiat',
+            'order_by_attributes': ['name', 'symbol'],
+            'ascending': [True, False],
```

Comment:
Can you check for the case where those lists don't have the same length? Or tell me if you know?